src/xml.js

```diff
@@ -126,9 +126,9 @@ define([
             return value; // value contains no character entity references or empty tags.
         }
         var xml = normalize(value, inner),
-            refs = /(?:&lt;(=?\s)|(\s)&gt;|&amp;(\s))/g;
+            refs = /(?:&lt;(=?\s)|(\s)&gt;|&amp;)/g;
```

Comment:
Just want to question if this change is indeed safe. The whitespace captures looked random, but it seems likely that the reason that we're matching whitespace after '<', whitespace before '>', and whitespace after '&' is specifically because that means those symbols aren't being used maliciously. '&amp;'. I'd imagine that the above code means that 'A>B' comes out mangled, but intentionally so. Do we risk anything by allowing '&lt;' type tokens into the strings?

Reply:
I can confirm that the the '<' and '>' characters without whitespaces do show up escaped in the question tree (A&gt;B for the example you described), though I'm not entirely sure if displaying them in that form pose any risks.

Reply:
I'm not familiar enough with this code to know why Daniel felt that we shouldn't escape '&' if used as a tag -- it might make sense to wait for his feedback. If we commit to this change as is, however, I think the function comment also needs to change. It claims:
> Un-escapes &, < and > when not used as tag delimiters...

which is no longer the case for ampersand with this change.

Reply:
I'm leaning toward @mjriley's concern that this may not be safe. I think I required a space so as to not double-escape character entity references like `&lt;`. However, it does not appear that I included any tests to verify that those references pass through unescaped (I would have expected such a test to fail with the change you have made here). I'm having a hard time evaluating if this will mangle any pre-existing forms if they are loaded and then saved by the form builder? It may be worthwhile to try to come up with and test a scenario where a character entity reference is expected to pass through.

> If we commit to this change as is, however, I think the function comment also needs to change. It claims:
>
>>    Un-escapes &, < and > when not used as tag delimiters...
>
> which is no longer the case for ampersand with this change.

+1

Reply:
Discussed in meeting with Minha (abbreviated):

- This regex/replacement should not touch or change `&amp;amp;`. A test should be added to assert this invariant.
- The fix for that is something like
```suggestion
            refs = /(?:&lt;(=?\s)|(\s)&gt;|&amp;([^\s];|\s)/g;
```
And then below the `("&" + amp)` expression would change to something like
`(amp.endsWith(";") ? match : ("&" + amp))`